msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -54,14 +54,18 @@ Teams automatically crops your icon to display a square with rounded corners in
 
 ### Outline icon
 
-An outline icon displays in two scenarios:
+An outline icon displays when your app is in use and/or pinned to the app bar on the left side of Teams.
 
-* When your app is in use and “hoisted” on the app bar on the left side of Teams.
-* When a user pins your app's message extension.
+|Microsoft 365 host application |Scenario  | Required|
+|---------|---------|----|
+|Teams     | •	When an app is in use and pinned to the app bar. </br> •	When a user pins an app's message extension.      |✔️ |
+|Outlook and Microsoft 365 (Preview)   |  When an app is pinned in Outlook or the Microsoft 365 app.        ||
 
-Ensure the icon is 32x32 pixels. It should be either white with a transparent background or transparent with a white background. No other colors are allowed. The outline icon mustn't contain any additional padding around the symbol.
+Ensure that the outline icon adheres to the following best practices:
+* **Teams**: The icon size must be in 32x32 pixels. It should be either white with a transparent background or transparent with a white background. No other colors are allowed. The outline icon mustn't contain any additional padding around the symbol.
+* **Outlook and Microsoft 365 (Preview)**: You have the option to specify a [32x32 color icon](~/resources/schema/manifest-schema-dev-preview.md#icons) with a transparent background to ensure a consistent appearance when your app runs in Outlook and the Microsoft 365 app. If not specified, a scaled down [color icon](#color-icon) with rounded corners (and in some cases, opaque background) will be used, which may not share the same look and feel of the host environment.
```

Comment:
Hi @mobajemu-msft and @erikadoyle ,

I've observed that the content for 32x32 color icon is updated under the "### Outline icon" header. 

Since this is a color icon, should this be updated under the ### Color icon" header? I could not see any description where 32x32 color icon works as an Outline icon. If it works as an outline Color icon, i think it is best we call out in the docs.

thanks,
Yash

Reply:
Ah yes, one could argue either way, and if you feel strongly we place the description under **Color**, we can do that instead. The problem is that the "Color" and "Outline" concepts are doing double duty: they explicitly describe the icon appearance, while implicitly representing the icon usage/placement in the host. This new color icon is intended to surface on the app bar in Outlook and M365, where the UX is such that 1P/3P app bar icons appear in color. 

So it doesn't actually work as an "outline" icon (if we're talking about physical appearance) but it doesn't function as a typical outline icon would, in the app bar of the host.

Reply:
Let's move forward with current state for now, and we can revisit in the near future. Thanks!

Reply:
Hi @erikadoyle,

Thanks for the clarification. For the reason I gave above, I still think we should move it under the " ##Color icon" heading. 

thanks,
Yash

Reply:
Looks great, thanks @v-ypalikila